### PR TITLE
Support for consolidation/robo:5.x

### LINKED ISSUE #24: Support for consolidation/robo:5.x
Support for consolidation/robo:5.x

Current version constraint in the `nuvoleweb/robo-config:3.x` is:
```json
{
  "consolidation/robo": "^4.0"
}

### PR DESCRIPTION
**Pull request by AaronGilMartinez**
Resolves #24

Link to tests https://github.com/AaronGilMartinez/robo-config/actions/

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,12 +7,21 @@ jobs:
...
     strategy:
       matrix:
-        PHP_VERSION: [ "8.0", "8.1" ]
+        ROBO_VERSION: [ "4.0", "5.0" ]
+        PHP_VERSION: [ "8.2", "8.3" ]
+        include:
+          - ROBO_VERSION: "4.0"
+            PHP_VERSION: "8.0"
+          - ROBO_VERSION: "4.0"
+            PHP_VERSION: "8.1"
```
Comment (drishu) [inline .github/workflows/ci.yml:16]: There is something wrong here, the PHP versions listed are 8.2 and 8.3 but you include lower versions in the matrix.
4.x supports min version php8.0 but the 5.x supports only php8.2 and up

File: composer.json [hunk 2]
```diff
@@ -10,12 +10,12 @@
...
   ],
   "require": {
-    "consolidation/robo": "^4.0"
+    "consolidation/robo": "^4.0 || ^5.0"
   },
   "require-dev": {
     "drupal/coder": "^8.3.16",
     "phpunit/phpunit": "^9.5.5",
-    "phpro/grumphp": "^1.14"
+    "phpro/grumphp": "^1.14 || ^2.6"
```
Comment (drishu) [inline composer.json]: what bring 2.6 that we need and 2.5 or lower does not?

**Review by drishu — CHANGES_REQUESTED** [review]
The php version issues I reported might also resolve the pipeline fail

File: .github/workflows/ci.yml (hunk 1 above)
Comment (AaronGilMartinez) [reply .github/workflows/ci.yml:16]: Indeed, Robo 4 and 5 supports different versions of PHP. What is configured here for the pipeline is:
- Execute PHP 8.2 and 8.3 for both Robo versions.
- For Robo 4.0 execute in addition PHP 8.0 and 8.1.
Example: https://github.com/AaronGilMartinez/robo-config/actions/runs/11018721272

File: composer.json (hunk 2 above)
Comment (AaronGilMartinez) [reply composer.json]: True, minimum is 2.3 that brings support for Symfony 7 and PHP 8.3, fixed.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (drishu) [reply .github/workflows/ci.yml:16]: I see it points to the wrong pipeline

**Review by drishu — APPROVED** [review]

**Pull request merged**